automation/test-linux.sh

```diff
@@ -63,48 +69,60 @@ fi
 
 delete_vm(){
   vm_name=$1
-  local template_name=$2
+
   set +e
   #stop vm
   ./virtctl stop $vm_name -n $namespace
   #delete vm
-  oc process -n $namespace -o json $template_name NAME=$vm_name SRC_PVC_NAME=$TARGET-datavolume-original SRC_PVC_NAMESPACE=kubevirt | \
-    oc delete -n $namespace -f -
+  oc process $2 -n $namespace -o json NAME=$vm_name SRC_PVC_NAME=$TARGET-datavolume-original SRC_PVC_NAMESPACE=kubevirt | \
+    ${KUBE_CMD} delete -n $namespace -f -
   set -e
   #wait until vm is deleted
-  while oc get -n $namespace vmi $vm_name 2> >(grep "not found") ; do sleep $sample; done
+  while ${KUBE_CMD} get -n $namespace vmi $vm_name 2> >(grep "not found") ; do sleep $sample; done
 }
 
 run_vm(){
   vm_name=$1
   template_path="dist/templates/$vm_name.yaml"
-  local template_name=$( oc get -n ${namespace} -f ${template_path} -o=custom-columns=NAME:.metadata.name --no-headers -n kubevirt )
+  local template_option
   running=false
 
-  set +e
+  # add cpumanager=true label to all worker nodes
+  # to allow execution of tests using high performance profiles
+  ${KUBE_CMD} label nodes -l kubevirt.io/schedulable cpumanager=true --overwrite
```

Comment:
can you delete this line? It is already in test.sh file on line 145

Reply:
Done